modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -27,6 +27,6 @@ FTP, FTPS, SFTP, JMS, SMTP, HL7 MLLP (only HL7 MLLP has its individual intro pag
 
 == Exchanges
 
-AS2, RNIF (each item has it’s individual intro page)
+AS2, RNIF (each item will have it’s individual intro page)
```

Comment:
1. MuleSoft style discourages the use of "will" -- please use present voice.
2. "it's" is a typo in this context and should be "its" (possessive).

Reply:
This brings up a good point. The items in parenthesis are indications of what will come with the docs. So, this will be gone for Beta. I guess that I should not use the future tense even for indicating items.